### DISCUSSION TITLE
One license for multiple accounts

### DISCUSSION THREAD

**Discussion by csilk** [burkybang/SuiteAdvanced-Feedback#105] [Q&A] [answered]
I use multiple email addresses (multiple accounts) to access NetSuite. Do I need one SuiteAdvanced license per email address?

**Comment by burkybang** [comment, marked as answer]
Nope, the email addresses used for NetSuite and SuiteAdvanced are completely unrelated. Use whatever email address you'd like for SuiteAdvanced (personal or work). It's only for license verification and for receiving receipts or payment.
